Tests/test_PDB.py

```diff
@@ -1576,6 +1576,20 @@ def test_pdb_to_xyzr(self):
         assert len(msms_radii) == len(biopy_radii)
         self.assertSequenceEqual(msms_radii, biopy_radii)
 
+class FragmentMapperTests(unittest.TestCase):
+    """Tests for FragmentMapper module."""
+    
+    def self_test(self):
+        """Self test for FragmentMapper module."""
+        p = PDBParser()
+        pdb1= "PDB/1A8O.pdb"
+        s = p.get_structure("X", pdb1)
+        self.assertEqual(m, s[0])
+        fm = FragmentMapper(m, 10, 5, "levitt_data")
+        for r in Selection.unfold_entities(m, "R"):
+            self.assertEqual(str(r), "R")
+            if r in fm:
+                print(fm[r])
```

Comment:
Give me a minute, I will have to run the script to check the expected value for the variable.